examples/beta/azure-container-apps-arm/README.md

```diff
@@ -0,0 +1,127 @@
+4. Fill out the following fields:
+   - **Subscription** : Choose the subscription you prefer.
+   - **Resource group**: Choose an existing Resource Group or create a new one using **Create new** button.
+   - **Region**: Choose the region you prefer.
+   - **Container App Name**: Enter a name you'd like to use, default will be `op-connect-con-app`.
+   - **Container App Env Name**: Enter a name you'd like to use, default will be `op-connect-con-env`.
+   - **Container App Log Analytics Name**: Enter a name you'd like to use, default will be `op-connect-con-app-log-analytics`.
+   - **Credentials** : Paste the entire contents of the `1password-credentials.json` file.
+5. Click **Review + create**.
+6. Once the validation succeeds, click **Create**. It is expected to take a couple of minutes to complete the deployment.
+
+## Step 3: Test your Connect Server
+
+Once your deployment is complete, click **Go to resource group** and click on the container app you created.
+
+To test if your Connect Server is online, choose **Overview** in your application's sidebar, then copy your **Application Url** link. This is your **Connect Server URL**. Use your 1Password Connect access token to interact with the API (e.g., via a `curl` request or a client application) to verify that your Connect Server is connected to your 1Password account.
+
+```sh
+curl --silent --show-error --request GET --header "Accept: application/json" \
+  --header "Authorization: Bearer mF_9.B5f-4.1JqM" \
+  https://op-connect.example.com/v1/vaults
+```
+
+<details>
+<summary>Example JSON response:</summary>
+
+```json
+[
+  {
+    "attributeVersion": 1,
+    "contentVersion": 9,
+    "createdAt": "2025-03-21T13:06:35Z",
+    "id": "hgm3sn37vkj3lsdkeouq46wyca",
+    "items": 5,
+    "name": "REDACTED",
+    "type": "USER_CREATED",
+    "updatedAt": "2025-04-18T16:38:40Z"
+  },
+  {
+    "attributeVersion": 1,
+    "contentVersion": 2,
+    "createdAt": "2025-03-21T02:15:33Z",
+    "id": "mn4w65hq6ar7nc2fc7qgoru3ki",
+    "items": 1,
+    "name": "REDACTED",
+    "type": "USER_CREATED",
+    "updatedAt": "2025-03-21T02:16:32Z"
+  },
+  {
+    "attributeVersion": 1,
+    "contentVersion": 94,
+    "createdAt": "2025-03-21T15:20:41Z",
+    "id": "z4s56cbejab6q6urod3l5icqky",
+    "items": 3,
+    "name": "REDACTED",
+    "type": "USER_CREATED",
+    "updatedAt": "2025-04-29T02:49:13Z"
+  }
+]
+```
+
+</details>
+<br />
+
+## Step 4: Integrate with your application
+
+To use the Connect Server, configure your application to communicate with the Connect Server URL using the access token provided in your 1Password account. Follow the [1Password Connect integration guide](https://developer.1password.com/docs/connect) to set up your application.
+
+<hr>
+
+## Update your Connect Server in the Azure Portal
+
+> [!TIP]
+> Check for 1Password Connect Server updates on the [1Password releases page](https://releases.1password.com/).
```

Comment:
Connect doesn't have releases here. The only place for Connect releases is [the changelogs in this repo](https://github.com/1Password/connect/blob/main/CHANGELOG.md).